alphastats/DataSet_Preprocess.py

```diff
@@ -430,6 +434,9 @@ def preprocess(
                 self.mat, self.metadata, self.sample, self.preprocessing_info
             )
 
+        if replace_zero:
```

Comment:
To be honest I still don't quite get why we don't unify zeros with nans

Reply:
There are instances, where the uploaded data might include real 0 values. For example in one of my own usecases where I would express profiles as percentages and then impute with 0. This would happen outside upfront and be a custom upload with real 0s.